syntax = "proto3";

message MessageTwo
{
    string mcMessageString=1;
    int64 mnTime=2;
}
